charts/everest/crds/everest.yaml

```diff
@@ -517,6 +517,933 @@ spec:
               engine:
                 description: Engine is the database engine specification
                 properties:
+                  affinity:
+                    description: Affinity defines scheduling affinity.
+                    properties:
```

Comment:
> Should we bump the chart version?

That happens automatically when you run the action for publishing the chart

Reply:
btw, let's add the same rule we have in other repos - block the PR if there are unresolved comments. wdyt?

Reply:
How it works is you merge everything to `main`, which is version 0.0.0, and never publish this.

Then you create a new chart release (like 1.4.0), and it updates the chart version and publish the chart

Reply:
> btw, let's add the same rule we have in other repos - block the PR if there are unresolved comments. wdyt?

oh, we don't own this repo. but it could be beneficial for others too :)

Reply:
@mayankshah1607 we should bump the chart version, right?

Reply:
> @mayankshah1607 we should bump the chart version, right?

No no, not for the main chart. We commit everything to `0.0.0` (`main` branch), and then bump the version at the time of releasing the chart (that is done by the CI action)

Reply:
Sorry I commented without refreshing the page and didn't see your comments.